types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,191 @@ declare namespace googleMapReact {
+            BOTTOM_LEFT: number;
+            BOTTOM_CENTER: number;
+            BOTTOM: number;
+            BOTTOM_RIGHT: number;
+            CENTER: number;
+        };
+        DirectionsStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            MAX_WAYPOINTS_EXCEEDED: string;
+            NOT_FOUND: string;
+        };
+        DirectionsTravelMode: {
+            DRIVING: string;
+            WALKING: string;
+            BICYCLING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
+        };
+        DirectionsUnitSystem: {
+            METRIC: number;
+            IMPERIAL: number;
+        };
+        DistanceMatrixStatus: {
+            OK: string;
+            INVALID_REQUEST: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            UNKNOWN_ERROR: string;
+            MAX_ELEMENTS_EXCEEDED: string;
+            MAX_DIMENSIONS_EXCEEDED: string;
+        };
+        DistanceMatrixElementStatus: {
+            OK: string;
+            NOT_FOUND: string;
+            ZERO_RESULTS: string;
+        };
+        ElevationStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            yz: string;
+        };
+        GeocoderLocationType: {
+            ROOFTOP: string;
+            RANGE_INTERPOLATED: string;
+            GEOMETRIC_CENTER: string;
+            APPROXIMATE: string;
+        };
+        GeocoderStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            ERROR: string;
+        };
+        KmlLayerStatus: {
+            UNKNOWN: string;
+            OK: string;
+            INVALID_REQUEST: string;
+            DOCUMENT_NOT_FOUND: string;
+            FETCH_ERROR: string;
+            INVALID_DOCUMENT: string;
+            DOCUMENT_TOO_LARGE: string;
+            LIMITS_EXCEEDED: string;
+            TIMED_OUT: string;
+        };
+        MapTypeControlStyle: {
+            DEFAULT: number;
+            HORIZONTAL_BAR: number;
+            DROPDOWN_MENU: number;
+            INSET: number;
+            INSET_LARGE: number;
+        };
+        MapTypeId: {
+            ROADMAP: string;
+            SATELLITE: string;
+            HYBRID: string;
+            TERRAIN: string;
+        };
+        MaxZoomStatus: {
+            OK: string;
+            ERROR: string;
+        };
+        NavigationControlStyle: {
+            DEFAULT: number;
+            SMALL: number;
+            ANDROID: number;
+            ZOOM_PAN: number;
+            Vz: number;
+            Ot: number;
+        };
```

Comment:
Could not find `NavigationControlStyle` in [the documentation](https://developers.google.com/maps/documentation/javascript/reference). Where does it come from ?

Reply:
Please see [main comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60329#issuecomment-1126798899) for response on where it comes from. 

The only mention I can find of this is [here](https://developers.google.com/maps/documentation/javascript/releases?hl=en#feb-26,-2010) where it mentions that the ZOOM_PAN option was not working in IE8 back in 2010, but there are no other mentions of this option in the documentation at all (even that it's deprecated, or been removed from the docs!)